spec/features/users/current_user_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+require "./app/models/user"
+
+describe "current_user" do
+  scenario "can't sign in again" do
+    user = User.create(name: "John Doe",
+                       email: "user@gmail.com",
```

Comment:
Align to two spaces tab on the next line. Fore example:

```rb
user = User.create(name: "John",
  email: "john@example.com",
)
```

On the other hand, use a factory here instead.